compose/material/material/api/desktop/material.api

```diff
@@ -496,7 +496,7 @@ public final class androidx/compose/material/ExposedDropdownMenuDefaults {
 	public final fun textFieldColors-DlUQjxs (JJJJJJJJJJJJJJJJJJJJJJLandroidx/compose/runtime/Composer;IIII)Landroidx/compose/material/TextFieldColors;
 }
 
-public final class androidx/compose/material/ExposedDropdownMenuKt {
+public final class androidx/compose/material/ExposedDropdownMenu_skikoKt {
```

Comment:
I would suggest to keep the name `ExposedDropdownMenuKt`. It would be easier to upstream. Otherwise, when this API becomes stable, and we accidentally won't notice that (there won't be a merge conflict), we will stuck with a stable `ExposedDropdownMenu_skikoKt`, which we have to keep "hidden deprecated" after renaming back to `ExposedDropdownMenu`

Reply:
Disagree. There is a related thread, let's discuss separately. Currently, the safest option is to revert it to 1.7 state

Reply:
https://youtrack.jetbrains.com/issue/CMP-7502 for tracking

Reply:
Okay, didn't know that it was changed after 1.7. Rechecked, and 1.7 had [skikoKt](https://github.com/JetBrains/compose-multiplatform-core/blob/6a03ba934d30f31bf55c36d36e46aec9a7701477/compose/material/material/api/desktop/material.api#L516C46-L516C73)